backend/blueprint/example.yaml

```diff
@@ -0,0 +1,17 @@
+runtime: python
+env: flex
+entrypoint: gunicorn -b :$PORT main:app
```

Comment:
May be we don't need this as docker will handle and take care of the same. Also I didn't understand that you are creating a custom entry and still using the base python config in the documentation

Reply:
@yashLadha Oops! I forgot to remove. Removing

Reply:
Seems you are forgetting alot